codegen/templates/tchannel_client.tmpl

```diff
@@ -161,6 +159,9 @@ type {{$clientName}} struct {
 			return respHeaders, err
 		{{else -}}
 			resp, err = {{.GenCodePkgName}}.{{title $svc.Name}}_{{title .Name}}_Helper.UnwrapResponse(&result)
+			if err != nil {
+				logger.Error("Unable to unwrap {{$clientName}}.{{$methodName}} ({{$serviceMethod}}) response", zap.Error(err))
```

Comment:
We do not want a dynamic log string here. Put them all in fields.

Reply:
Done.